README.md

```diff
@@ -23,10 +23,10 @@ Register has been used to test purchasing flows of our [Flagship Reader App](htt
 
 ### Overview
 
-Register is a library and companion app that allows seamless mocking of responses from Google Play's In-app Billing. 
-Register works by implementing the same interface as Google's In-app Billing library [InApp Billing Service](https://github.com/googlesamples/android-play-billing/blob/master/TrivialDrive/app/src/main/aidl/com/android/vending/billing/IInAppBillingService.aidl).
+Register is a companion app and library that allows seamless mocking of responses from Google Play Store Billing. 
+Register works by implementing the same interface as the Google Play Store Billing library [Google Play Billing](https://developer.android.com/google/play/billing/billing_overview).
```

Comment:
I think it also makes sense to link to Google's updated sample repo (ClassyTaxi)